src/_locales/uk/messages.json

```diff
@@ -545,15 +545,15 @@
         "description": ""
     },
     "share_button_title_facebook": {
-        "message": "Поширити на Facebook",
+        "message": "Поширити на Фейсбуці",
```

Comment:
It's better keep the brand names like Facebook, Firefox, Mozilla, Twitter untranslated